docs/design/content-add-ins.md

```diff
@@ -41,11 +41,51 @@ For Mac, the personality menu measures 26x26 pixels, but floats 8 pixels in from
 
 ## Implementation
 
-There are minor differences in the manifests between content add-ins and add-ins that use task panes.
+There are minor differences in the manifests between content add-ins and add-ins that use task panes. Open the tab for the type of manifest you are using.
+
+# [Unified manifest for Microsoft 365](#tab/jsonmanifest)
+
+> [!NOTE]
+> The unified manifest is available in Excel, PowerPoint, and Word as a developer preview. For Outlook, it is generally available and can be used in production add-ins. 
+
+Configure the manifest with the following steps:
+
+1. Add a "contentRuntimes" child array to the extension object in the "extensions" array.
+1. Add an anonymous content runtime object in the "contentRuntimes" array.
+1. Set the "id" property of the object to a descriptive name.
+1. Set the "code.page" object to the full URL of the page that contains the content.
+1. Optionally, set the "requestedWidth" and "requestedHeight" properties to a size between 32 and 1000 pixels. If these properties are not used, the Office application determines the size.
+1. Optionally, set the the "disableSnapshot" property to `true` to prevent Office from saving a snapshot of the content component with the document. 
```

Comment:
```suggestion
1. Optionally, set the "disableSnapshot" property to `true` to prevent Office from saving a snapshot of the content component with the document. 
```